src/commands/edge_app.rs

```diff
@@ -339,7 +339,7 @@ impl EdgeAppCommand {
         let revision = self.get_latest_revision(actual_app_id).unwrap_or(0);
 
         let remote_files = self.get_version_asset_signatures(actual_app_id, revision)?;
-        let changed_files = detect_changed_files(&local_files, &remote_files)?;
+        let mut changed_files = detect_changed_files(&local_files, &remote_files)?;
```

Comment:
Does it have to be mut ? I can't understand from name why it has to be and other parameter is Const. can you return the new updated list instead ?

Reply:
There is a new property copied in the FileChanges, that is to be updated later after the call to copy_assets. I can move it out from FileChanges if you want to make it cleaner.

Reply:
I think it does so if you could and it does not take too much effort please do